src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -56,7 +56,9 @@
    (cond
      ;; TODO: use account-switcher when available (issue #16456)
      (and support-account-switcher? (= content :account-switcher))
-     [rn/view {:style style/account-switcher-placeholder}]
+     [rn/touchable-opacity
+      {:style    style/account-switcher-placeholder
+       :on-press #(js/alert "Not implemented yet")}]
```

Comment:
We agreed to use `pressable` instead of other touchable components.

@ilmotta should we add this to the guidelines? I see `touchable-opacity` still often being used in new PRs